xml/file_management.xml

```diff
@@ -204,6 +204,65 @@
 
  </sect1>
 
+ <sect1 xml:id="sec-sec-prot-general-home-permissions">
+  <title>Changing home directory permissions from 775 to 700</title>
```

Comment:
from 755 to 700

Reply:
Wow, good catch! This has been incorrect all the way, in the bug, your draft and the existing best practice at
https://documentation.suse.com/sbp/all/html/OS_Security_Hardening_Guide_for_SAP_HANA_SLES15/index.html#id-changing-home-directory-permissions-from-775-to-700
Filed as https://github.com/SUSE/suse-best-practices/issues/228, I'll take care of fixing it.